### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by ejether**
Third party naive edit of getting-started.md

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -14,21 +14,21 @@ After running `pentagon start-project` you will have a directory with a layout s
...
+`config/private`, in addition to `secrets.yml` also contains SSH keys generated by `start-project`. Unless you opted to not create the keys, the `admin-vpn` key pair will be uploded to AWS for you when the VPN instance is created and the `*-kube` keys will automatically be uploaded when `kops` is invoked to create the Kubernetes cluster. The other keys, `production-private`, `working-private` are created as a convenience to be used for any instances that are created in the VPC `private-working` and `private-production` subnets. When `kops` is invoked to create the cluster, the Kubernetes config secret will also be created as `config/private/kube_config`
```
Comment (geojaz) [inline docs/overview.md]: s/uploded/uploaded

File: docs/overview.md [hunk 2]
```diff
@@ -14,21 +14,21 @@ After running `pentagon start-project` you will have a directory with a layout s
...
 
 ### default/
 The `default/` contains most of the moving parts of the infrastructure repository. The name `default` is not important! The contents are. The goal is that the contents of the `default` directory can be deep copied and create parallel (cloud provider, cloud account, vpc) infrastructure in a single repository. *Consider this a guidline, not a rule!*
```
Comment (geojaz) [inline docs/overview.md]: s/guidline/guideline

File: docs/overview.md [hunk 3]
```diff
@@ -52,9 +52,9 @@ Contains `working/` and `production/` directories. Both are laid out identically
 ├── resources
 └── vars.sh
 ```
-`cluster-config/kops.sh` is a bash script that creates the kops `cluster.spec` and uploads it to the s3 bucket set in `default/account/vars.sh`. 
+`cluster-config/kops.sh` is a bash script that creates the Kops `cluster.spec` and uploads it to the S3 bucket set in `default/account/vars.sh`.
```
Comment (geojaz) [inline docs/overview.md]: kops is the correct spelling.

Comment (ejether) [reply docs/overview.md]: 👍

**Pull request merged**